coordinator/src/test/scala/filodb.coordinator/queryplanner/PlannerHierarchySpec.scala

```diff
@@ -2818,14 +2789,18 @@ class PlannerHierarchySpec extends AnyFunSpec with Matchers with PlanValidationS
       partitionLocationProvider, singlePartitionPlanner, "local",
       MetricsTestData.timeseriesDataset, queryConfig
     )
-    for (query <- queries) {
-      val lp = Parser.queryRangeToLogicalPlan(query, TimeStepParams(startSec, stepSec, endSec))
-      val promQlQueryParams = PromQlQueryParams(query, startSec, stepSec, endSec)
-      assertThrows[BadQueryException] {
-        engine.materialize(lp, QueryContext(origQueryParams = promQlQueryParams,
-          plannerParams = PlannerParams(processMultiPartition = true)))
-      }
-    }
+    val lp = Parser.queryRangeToLogicalPlan(query, TimeStepParams(startSec, stepSec, endSec))
+    val promQlQueryParams = PromQlQueryParams(query, startSec, stepSec, endSec)
+    val execPlan = engine.materialize(lp,
+      QueryContext(origQueryParams = promQlQueryParams,
+        plannerParams = PlannerParams(processMultiPartition = true))
+    )
+    val expectedPlan =
```

Comment:
Could you try a query like:
```
sum(rate(test{_ws_ = "demo", _ns_ = "localNs"}[1h]))
+
sum(rate(test{_ws_ = "demo", _ns_ = "localNs"}[1h] offset 30m))
```

The left/right of the split are stitched correctly together, but we're looking for one additional stitch, as well, since some datapoints cannot be computed without values from both sides of the split.